libraries/libfc/src/crypto/bls_private_key.cpp

```diff
@@ -25,38 +25,49 @@ namespace fc::crypto::blslib {
       return bls_private_key(v);
    }
 
-   template<typename Data>
-   Data from_wif( const string& wif_key )
+   string to_wif(vector<uint8_t> secret, const fc::yield_function_t& yield )
    {
-   /*   auto wif_bytes = from_base58(wif_key);
+      FC_ASSERT(secret.size() == 32);
+
+      std::array<uint8_t, 32> v2;
+
+      std::copy(secret.begin(), secret.end(), v2.begin());

```

Comment:
I don't see any reason to make this copy. Should be able to use `secret` vector for the `memcpy`.